src/Tools/ConfigurationSchemaGenerator/ConfigSchemaEmitter.cs

```diff
@@ -25,6 +26,9 @@ internal sealed partial class ConfigSchemaEmitter(SchemaGenerationSpec spec, Com
     [GeneratedRegex(@"( *)\r?\n( *)")]
     private static partial Regex Indentation();
 
+    [GeneratedRegex(@"(?<namespace>[A-Z]):(?<actual>[a-zA-Z0-9.]+)")]
```

Comment:
.NET types can contain non-ASCII characters too. Perhaps the `actual` pattern should just be `\w+`.

nit: the first character isn't the namespace, but rather the member type.

Reply:
> Perhaps the actual pattern should just be `\w+`.

Actually I think that's insufficient. I was trying to work out how this changed, for example:

```diff
- extracting the hash out\nof the T:Azure.Response`1 and validating the hash themselves.
+ extracting the hash out\nof the 'Azure.Response`1' and validating the hash themselves.
```

In this case, both `.` and `` ` `` wouldn't match `[a-zA-Z0-9.]` or `\w+`.

It works because the pattern is not validating string starts and string ends. So the pattern would match all of these, which I don't think is what's intended:

- `AAAAA:AAAAAA`
- `(*&(*& P:Hello (*&(*&`
- `      A:P     `

None of these would produce the correct result when the first two characters are removed.

Instead, you likely want to use `$` and `^` to mark the start and end of the strings.